sklearn/utils/multiclass.py

```diff
@@ -344,11 +344,33 @@ def _check_partial_fit_first_call(clf, classes=None):
 
         else:
             # This is the first call to partial_fit
+            if issparse(y):
+                raise ValueError("Sparse y is not supported.")
+
+            # Validate classes against unique labels in y
```

Comment:
Is this for the multi-output case? I which estimator is this used? Only trees have multi-output and they don't have a ``partial_fit``, right?